### DISCUSSION TITLE
Search box seem to be wrongly styled in react/tailwind project but maybe I am doing it wrong...

### DISCUSSION THREAD

**Discussion by juliusjoosten** [scalar/scalar#973] [Q&A] [answered]
Hi,

so far I am happy how this project looks like, however it seems that the search box isnt looking the way it should. I am using firefox (edit: looks the same in chromium).
![Screenshot from 2024-02-09 08-33-48](https://github.com/scalar/scalar/assets/11553694/272ac79c-6cc4-4496-9cb7-cad25889c7a6)

I am not sure if its a bug or some side effect of the rest of my page. I am using veaury as described to make scalar work with the react stuff. Below is the code how I use the ApiReference.

`const ApiDocumentationContainer = ({ specUrl }) => {
	return (
		<div className="prefix-px-8">
			<ApiReference configuration={{ spec: {url: specUrl} }} />
		</div>
	)
}`

This is the span in the browser.
![Screenshot from 2024-02-09 08-42-37](https://github.com/scalar/scalar/assets/11553694/2b8d41ba-938d-4e68-855f-08da721b320c)

Maybe someone has an idea what happens to the search field...please let me know if further information is required.

**Comment by hanspagel** [comment, marked as answer]
Oops, I’ve create a issue: https://github.com/scalar/scalar/issues/975
